go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/kserve/kserve v0.14.0
 	github.com/kuadrant/authorino v0.18.1
-	github.com/kubeflow/model-registry v0.2.11
+	github.com/kubeflow/model-registry v0.2.7-alpha.0.20250210125605-b5509328e038
```

Comment:
why downgrading the version?

Reply:
This comes from a commit rather than a tag, specifically:

```shell
go get github.com/kubeflow/model-registry@b5509328e038700bb186a554858ba7b55c58a22a
```

If this concerns you, we can do an ad hoc release upstream and point to the new tag here

Reply:
so, are you going to update it?

Reply:
done